product_docs/docs/pgd/5/manual/04-installing-software.mdx

```diff
@@ -0,0 +1,249 @@
+```
+
+It will append
+
+```
+host all all all md5
+host replication all all md5
+```
+
+to `pg_hba.conf` which will enable the nodes to replicate.
+
+#### Enable authentication between nodes
+
+As part of the process of connecting nodes for replication, PGD logs into other nodes. 
+It will perform that log in as the user that Postgres is running under.
+For epas, this is the `enterprisedb` user.
+That user will need credentials to log into the other nodes.
+We will supply these credentials using the `.pgpass` file which needs to reside in the user's home directory.
+The home directory for `enterprisedb` is `/var/lib/edb`.
+
+Run this command to create the file:
+
+```
+echo -e "*:*:*:enterprisedb:secret" | sudo -u enterprisedb tee /var/lib/edb/.pgpass; sudo chmod 0600 /var/lib/edb/.pgpass
+```
+
+### Restart the server
+
+After all these configuration changes, it is recommended that the server is restarted with:
+
+```
+sudo systemctl restart edb-as-16
+```
+
+### Create the replicated database
+
+The server is now prepared for PGD. 
+We need to next create a database named `bdrdb` and install the bdr extension when logged into it.
+
+```
+sudo -u enterprisedb psql edb -c "CREATE DATABASE bdrdb"
+sudo -u enterprisedb psql bdrdb -c "CREATE EXTENSION bdr"
+```
+
+Finally, test the connection by logging into the server.
+
+```
+sudo -u enterprisedb psql bdrdb
+```
+
+You should be connected to the server. 
+
+
+
+## Summary: Installing PGD for EDB Postgres Advanced Server
+
+These are all the commands used in this section gathered together for your convenience.
+
+```
+sudo dnf -y install edb-bdr5-epas16 edb-pgd5-proxy edb-pgd5-cli
+sudo PGSETUP_INITDB_OPTIONS="-E UTF-8" /usr/edb/as16/bin/edb-as-16-setup initdb
+sudo systemctl start edb-as-16
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET shared_preload_libraries TO '\$libdir/bdr','\$libdir/dbms_pipe','\$libdir/edb_gen','\$libdir/dbms_aq'"
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET wal_level TO 'logical'"
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET track_commit_timestamp TO 'on'"
+sudo -u enterprisedb psql edb -c "ALTER USER enterprisedb WITH PASSWORD 'secret'"
+sudo -u enterprisedb psql edb -c "ALTER SYSTEM SET max_worker_processes TO '16'"
+echo -e "host all all all md5\nhost replication all all md5" | sudo tee -a /var/lib/edb/as16/data/pg_hba.conf
+echo -e "*:*:*:enterprisedb:secret" | sudo -u enterprisedb tee /var/lib/edb/.pgpass; sudo chmod 0600 /var/lib/edb/.pgpass
+sudo systemctl restart edb-as-16
+sudo -u enterprisedb psql edb -c "CREATE DATABASE bdrdb"
+sudo -u enterprisedb psql bdrdb -c "CREATE EXTENSION bdr"
+sudo -u enterprisedb psql bdrdb
+```
+
+## Other Postgres Installations
+
+### Installing PGD for EDB Postgres Extended Server
+
+If installing PGD with EDB Postgres Extended Server, there are a number of differences from the EPAS installation.
+
+* The BDR package to install is named `edb-bdr-pgextendedNN` (where NN is the PGE version number)
```

Comment:
The package has also the numer of bdr in its name, i.e. `edb-bdr5-pgextendedNN`

Reply:
Good spot, fixed.